src/devtools/client/debugger/src/utils/timings.js

```diff
@@ -6,6 +6,8 @@
 
 import zip from "lodash/zip";
 
+// Is this ever used?
```

Comment:
Left a comment so I could ask this at review - is this one of those "it's used by the test harness" situations?

Reply:
Does not look like it's being referenced anywhere, nor is it writing any values to the global (😅 )